packages/eslint-plugin-nx/src/rules/enforce-module-boundaries.ts

```diff
@@ -129,7 +130,9 @@ export default createESLintRule<Options, MessageIds>({
     }
 
     const npmScope = (global as any).npmScope;
-    const projectGraph = (global as any).projectGraph as ProjectGraph;
+    const projectGraph = mapProjectGraphFiles(
```

Comment:
I think remapping will happen multiple times, cause the object will remain when is used by the IDE for linting

Reply:
You are right, I will move it to the branch above.